getting-started.md

```diff
@@ -83,6 +83,54 @@ If everything works, you should see your starter app on your device or simulator
   </div>
 </div>
 
+## _Hot Reloading_ and _Full Application Restart_ from within IntelliJ
```

Comment:
Agreed, that was my feeling as well. That some of this content should be in the intellij-ide.md file. One recommendation: When I tried out the IntelliJ plugin, I missed that there was a separate page for the Flutter IntelliJ plugin. Doesn't it make sense to leave a link to that document in getting-started.md?